projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/MultiProgress.svelte

```diff
@@ -34,6 +34,8 @@
 					isLink={hasIssues}
 					class="jb-navigator-link"
 					to="/image-size-analysis/{group}/1"
+					trackEvent="clicked_isa_group_on_summary_page"
+					trackEventProps={group}
```

Comment:
I don't understand why this uses a different tracks event to the others on this page. Is there a distinction we need to track?

Reply:
Thanks, I forgot to update the others!